packages/core/src/utils/dom.ts

```diff
@@ -1,3 +1,48 @@
+import type { Operation } from '../types'
+
+export class VDom {
```

Comment:
I find the name is a bit confusing and I don't really understand what it's supposed to do.

Reply:
It is used as an abstraction layer to remove and restore child elements while preserving the order.

I agree that `VDom` is not a good name. I've merged this class into the `Drauu` class.